corehq/util/workbook_reading/tests/test_spreadsheets.py

```diff
@@ -40,53 +40,27 @@ def _assert_worksheets_equal(self, worksheet1, worksheet2):
 @run_on_all_adapters(SpreadsheetCellTypeTest)
 def test_xlsx_types(self, open_workbook, ext):
     with open_workbook(get_file('types', ext)) as workbook:
-        if ext == 'xlsx':
-            self.assert_workbooks_equal(
-                workbook,
-                Workbook(
-                    worksheets=[
-                        make_worksheet(title='Sheet1', rows=[
-                            ['String', 'Danny'],
-                            ['Date', date(1988, 7, 7)],
-                            ['Date Time', datetime(2016, 1, 1, 12, 0)],
-                            ['Time', time(12, 0)],
-                            ['Midnight', date(1899, 12, 30)],
-                            ['Int', 28],
-                            ['Int.0', 5],
-                            ['Float', 5.1],
-                            ['Bool-F', False],
```

Comment:
does this mean you are no longer testing these types?

Reply:
They are being tested - but this avoids repeating by using a ternary operator instead of an if/else block.